### PR TITLE
chore(dependencies): Upgrade enzyme to 3.1.0

### PR DESCRIPTION
**Pull request by smouillour**
**What is the problem this PR is trying to solve?**
We need to upgrade enzyme version to the version 3.1.0 for TMC

**What is the chosen solution to this problem?**
Change the version in version.js and fix all tests

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

- Url to migration documentation : https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md
- specific problems encountered
  - [Element referential identity is no longer preserved](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#element-referential-identity-is-no-longer-preserved)
  - exemple of fix : (sepcify html tag)
image

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/cmf-cqrs/package.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
     "babel-preset-react": "6.11.1",
     "classnames": "2.2.5",
     "codacy-coverage": "^2.0.2",
-    "enzyme": "^2.7.1",
+    "enzyme": "^3.1.0",
+    "enzyme-adapter-react-15": "^1.0.1",
```
Comment (jmfrancois) [inline packages/cmf-cqrs/package.json]: please add it to version.js

File: packages/components/src/Drawer/Drawer.test.js [hunk 2]
```diff
@@ -52,8 +52,8 @@ describe('Drawer', () => {
 		expect(wrapper).toMatchSnapshot();
 	});
 	it('should render cancelActionComponent', () => {
-		const wrapper = shallow(cancelActionComponent({}));
-		expect(wrapper.find('<button />')).toBeTruthy();
+		const wrapper = mount(cancelActionComponent({}));
+		expect(wrapper.find('button')).toBeTruthy();
```
Comment (jmfrancois) [inline packages/components/src/Drawer/Drawer.test.js]: :'(

File: output/forms.eslint.txt [hunk 3]
```diff
@@ -13,5 +13,11 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/forms/src/UIForm/fields/Radios/Radios.component.js
   23:7  error  Form controls using a label to identify them must be programmatically associated with the control using htmlFor  jsx-a11y/label-has-for
 
-✖ 3 problems (3 errors, 0 warnings)
+/home/travis/build/Talend/ui/packages/forms/src/UIForm/Message/Message.component.test.js
+  11:4  error  Expected indentation of 4 tab characters but found 3                                              react/jsx-indent-props
+  12:4  error  Expected indentation of 4 tab characters but found 3                                              react/jsx-indent-props
+  13:4  error  Expected indentation of 4 tab characters but found 3                                              react/jsx-indent-props
+  14:3  error  The closing bracket must be aligned with the line containing the opening tag (expected column 4)  react/jsx-closing-bracket-location
```
[the remark this answers is not in the document]
Comment (smouillour) [reply output/forms.eslint.txt]: @jmfrancois has fix those errors : https://github.com/Talend/ui/pull/907/commits/10c48eb568b2e8f68be486a54e9e0859e00cd27c

**Comment by jsomsanith** [comment]
WARNING : merge on 21 dec 2017, not before

**Pull request merged**